# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/ipc/PVRLayer.ipdl
# Commit: e1b0906509ef
# Full Hash: e1b0906509efa6433978b9f980ec639b7eafbc6d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix non-webgl CI tests. r=handyman
#   
#   * Revert some partial webgl+oop+vr code.
#   * More missing FuncScope.
#   * Fix compile errors.
# ==============================================================================

diff -r 7e2a2b1b416f -r e1b0906509ef gfx/vr/ipc/PVRLayer.ipdl
--- a/gfx/vr/ipc/PVRLayer.ipdl	Wed Jan 08 22:19:23 2020 +0000
+++ b/gfx/vr/ipc/PVRLayer.ipdl	Wed Jan 08 22:19:26 2020 +0000
@@ -7,7 +7,6 @@
 
 include LayersSurfaces;
 include protocol PVRManager;
-include protocol PWebGL;
 
 using mozilla::gfx::Rect from "mozilla/gfx/Rect.h";
 
@@ -19,11 +18,9 @@
   manager PVRManager;
 
 parent:
-  async SubmitFrame(PWebGL webgl, uint64_t aFrameId, uint64_t aLastFrameId,
+  async SubmitFrame(SurfaceDescriptor aTexture, uint64_t aFrameId,
                     Rect aLeftEyeRect, Rect aRightEyeRect);
 
-  async ClearSurfaces();
-  
   async Destroy();
 
 child: